browser/js/create-game/create-game.html

```diff
@@ -8,10 +8,11 @@
                     <div class="row">
                         <div class="col s3">
                         </div>
+                        <h3h6>{{comm}}</h6>
```

Comment:
Also this seems like debug code right? `comm` is an object, so I assume you were interpolating it simply to inspect it. If so, remember to keep it out of master.